modules/exploits/user_access_manager_reflected_xss_shell_upload.rb

```diff
@@ -0,0 +1,32 @@
+class Wpxf::Exploit::UserAccessManagerReflectedXssShellUpload < Wpxf::Module
+  include Wpxf::WordPress::ReflectedXss
+
+  def initialize
+    super
+
+    update_info(
+      name: 'User Access Manager <= 2.0.0 Reflected XSS Shell Upload',
+      author: [
+        'Paul Williams <phyushin[at]phyubox.com>' # WPXF module
```

Comment:
Can you add credit for the original disclosure in the author list? (i.e. DefenseCode)